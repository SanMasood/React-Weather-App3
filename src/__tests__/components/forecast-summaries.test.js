import React from 'react';
import {cleanup, render} from '@testing-library/react';
import ForecastSummaries from '../../components/forecast-summaries';

const forecasts = [
    {
        date: 1234,
        description: "Hazy",
        icon: "icon1",
        temperature: {
            max: 40,
        },
    }, 
    {
        date: 4321,
        description: "Foggy",
        icon: "icon2",
        temperature: {
            max: 45,
        },
    }
];

afterEach(cleanup);

describe("Forecast Summaries", () => {
    const forecasts = [
        {
            date: 1234,
            description: "Hazy",
            icon: "icon1",
            temperature: {
                max: 40,
            },
        }, 
        {
            date: 4321,
            description: "Foggy",
            icon: "icon2",
            temperature: {
                max: 45,
            },
        }

    ];
    it("renders correctly to match ForecastSummaries snapshot", () => {
        const { asFragment } = render(<ForecastSummaries forecasts={forecasts} />);
        expect(asFragment).toMatchSnapshot();

    });

    xit("renders the correct amount of ForecastSummary component props", () => {

    })
})