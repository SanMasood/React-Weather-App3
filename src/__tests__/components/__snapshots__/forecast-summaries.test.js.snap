// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forecast Summaries renders correctly to match ForecastSummaries snapshot 1`] = `[Function]`;
